tests/surface_attributes/classify_aspect_test.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import rasterio
+import pytest
+from pathlib import Path
+
+from eis_toolkit.exceptions import (
+    InvalidRasterBandException,
+    InvalidParameterValueException,
+)
+from eis_toolkit.surface_attributes.classification import classify_aspect
+from eis_toolkit.surface_attributes.parameters import first_order
+
+parent_dir = Path(__file__).parent
+raster_path_single = parent_dir.joinpath("../data/remote/small_raster.tif")
+raster_path_multi = parent_dir.joinpath("../data/remote/small_raster_multiband.tif")
+raster_path_nonsquared = parent_dir.joinpath("../data/remote/nonsquared_pixelsize_raster.tif")
+
+
+def test_aspect_classification():
+    
+    for num_classes in [8, 16]:
```

Comment:
You should use `pytest.mark.parametrize` to avoid the for-loop in the test as the loop members do not interact with each other. E.g.

~~~python
@pytest.mark.parametrize("num_classes", [8, 16])
def test_aspect_classification(num_classes: int):
    with rasterio.open(raster_path_single) as raster:
        ...
~~~